S23/pmirukula/Conversion.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.*;
+import java.util.Locale;
+
+public class Conversion {
+	public void conversionlogic() 
+	{
+	}
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		System.out.println("Please enter the word");
+		String str = br.readLine();
+		System.out.println("Please enter the language you would like to translate the word");
+		String str_lang = br.readLine();
+		PrintWriter out = new PrintWriter("oceans.txt"); 
+		String output = str.toLowerCase(new Locale(str_lang));
+		if(str_lang == " tr " || str_lang == "az")

```

Comment:
// we should use the appropriate method for string called equals. More explaination here https://www.geeksforgeeks.org/difference-between-and-equals-method-in-java/
		// there is a white space in " tr". It was made on purpose?

Reply:
No, it was a mistake. will change it!
Thank you.